pages/gallery/_.vue

```diff
@@ -54,22 +74,30 @@
           return {
             description: response.items[0].fields.description,
             images: response.items[0].fields.hasPart,
-            title: response.items[0].fields.name
+            title: response.items[0].fields.name,
+            canonicalURL: null
           };
         })
         .catch((e) => {
           error({ statusCode: 500, message: e.toString() });
         });
     },
+    mounted() {
+      this.canonicalURL = window.location.href.split(/\?|#/)[0];
```

Comment:
This results in the og:url meta tag having the value `null` when requested solely by SSR which may impede indexing. Move it into the `asyncData` method, using `route` from the Nuxt context instead of relying on the CSR-only window location.